svc/pkg/cluster/worker/src/workers/server_install/install_scripts/components/traefik.rs

```diff
@@ -282,3 +294,32 @@ pub async fn gg_static_config() -> GlobalResult<String> {
 
 	Ok(config)
 }
+
+pub fn gg_dynamic_config(datacenter_id: Uuid) -> GlobalResult<String> {
+	let domain_job = unwrap!(util::env::domain_job(), "dns not enabled");
+
+	let main = format!("{datacenter_id}.{domain_job}");
+
+	Ok(formatdoc!(
+		r#"
+		# Always returns 200 at /status
+		[http.routers.ok-status]
+			entryPoints = ["lb-80"]
+			rule = "Path(`/status`)"
```

Comment:
won't this override any `/status` endpoints for game servers? unless their priority takes precedence, in which case we should add a comment noting that.

would be good to restrict this to a specific host, regardless.